rts/lra/test/basic/src/test/java/io/narayana/lra/arquillian/resource/NestedParticipant.java

```diff
@@ -48,16 +47,33 @@
 
     public static final String ROOT_PATH = "/nested";
     public static final String ENLIST_PATH = "/enlist";
+    public static final String PATH = "path";
 
     @Inject
     LRAMetricService lraMetricService;
 
+    @LRA(end = false)
+    @GET
+    @Path(PATH)
+    @Produces({MediaType.TEXT_PLAIN})
+    public Response runWithNestedContext(@HeaderParam(LRA.LRA_HTTP_CONTEXT_HEADER) URI lraId,
+                           @HeaderParam(LRA.LRA_HTTP_PARENT_CONTEXT_HEADER) URI parentId) {
+        if (parentId == null || lraId == null) {
+            return Response.serverError().build();
```

Comment:
we should add a message that says which one was null. The error message in test says it is only parentId.

Reply:
I don't think so, this is a test suite in which we already have tests that check the presence of LRA_HTTP_CONTEXT_HEADER.

Reply:
true, this was more about this particular test only. It is unlikely it wouldn't be caught elsewhere.

Reply:
This is a test for nested contexts so LRA_HTTP_CONTEXT_HEADER is not relevant.